### ISSUE TITLE
[website] on/off toggle button is unlabeled

### ISSUE THREAD

**Issue by asadotzler** [mozilla/fx-private-relay#225]
Steps to reproduce: navigate to the on/off toggle button at https://relay.firefox.com/accounts/profile/ with a screen reader active

Results: screen reader announces "button" with no label

Expected results: screen reader announces something like "forward emails on switch"

https://scottaohara.github.io/a11y_styled_form_controls/src/toggle-button-switch/ has some information on making this kind of button accessible.
